java/src/jmri/jmrix/loconet/LnCommandStationType.java

```diff
@@ -368,6 +590,11 @@ protected enum LocoResetSupport {
         NO_LOCO_RESET_SUPPORT, SUPPORTS_LOCO_RESET_FUNCTION
     }
 
+    public enum CommandStationFracType {
+        CLOCKNONE,
+        CLOCK13BIT,
+        CLOCK15BIT
+    }
```

Comment:
I suggest this enum is renamed to `CommandStationClockFracType`. It makes it easier to understand what it's about.